### ISSUE TITLE
Animate forklift 

### ISSUE THREAD

**Issue by jonreis90** [AttainableEntertainment/WarehouseSimulator#9]
Add the animations for the forklift in game, make the folks lift up and down appropriately, make the back wheels turn and make all wheels and steering rotate.

**Comment by jonreis90** [comment]
Skeletal mesh is done, need to program vehicle and then eventually animate everything, low priority

**Comment by jonreis90** [comment]
#38 must be completed first
